### PR TITLE
WIP: Rearange Sass files

### PR DESCRIPTION
**Pull request by Nateowami**
I've rearranged the Sass files. There are a lot of changes here, and I'm not entirely certain I've gotten everything right. I'm opening this PR for the sake of review, hoping these changes can be looked at separate from the changes to come, since it will be far easier to review that way.

From the body of the commit message:

> You can now compile Sass with:
> ./node_modules/.bin/node-sass src/Site/ -o src/Site/ --recursive
> ./node_modules/.bin/node-sass src/angular-app/ -o src/angular-app/ --recursive
>
> Throw in the --watch flag for watching. Simplified build is to come.

If someone wants to test this out on another machine that would be great.

What still needs to be done:

- Making sure the Scripture Forge Sass is arranged mostly correctly
- Updating Gulp to run node-sass instead of using the gulp-sass wrapper. Right now `gulp sass` and `gulp sass:watch` will not work as expected.

---
This change is [Reviewable](https://reviewable.io/reviews/sillsdev/web-languageforge/92)

### CODE REVIEW

File: src/Site/views/languageforge/theme/default/sass/languageforge_default.scss [hunk 1]
```diff
@@ -2,13 +2,10 @@
 @import '_variables.scss';
 
 // Pull in Bootstrap files - by default this loads ALL features of Bootstrap which we may not need
-@import '../../../../../../../../vendor_bower/bootstrap/scss/bootstrap.scss';
+@import '../.././../../../../vendor_bower/bootstrap/scss/bootstrap.scss';
```
Comment (darcywong00) [inline src/Site/views/languageforge/theme/default/sass/languageforge_default.scss]: I thought we were removing this import.
and if there's a reason we're keeping this, there's a typo in the path (missing a '.')

Comment (Nateowami) [inline src/Site/views/languageforge/theme/default/sass/languageforge_default.scss]: You're right, it's a typo, though it's not a missing `.`, but rather a `./` should be removed (it works how it is, but it's redundant).

About removing the import, are you thinking of Font Awesome?

**Comment by megahirt** [comment]
:lgtm: I'm not going to pretend to have looked at all of these files.  Rather, I'm mostly interested in merging in this good work so that we can figure out if there's anything that's not working as expected.

---

Review status: 0 of 102 files reviewed at latest revision, 1 unresolved discussion, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/sillsdev/web-languageforge/92#-:-Kftv77z-6xkjOErb1h2:bijz48w)*

**Comment by megahirt** [comment]
Reviewed 53 of 68 files at r1, 88 of 88 files at r2.
Review status: all files reviewed at latest revision, 1 unresolved discussion, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/sillsdev/web-languageforge/92)*

**Pull request merged**